Cargo.toml

```diff
@@ -7,19 +7,25 @@ members = [
 	"dpdk-sysroot-helper",
 	"errno",
 	"id",
+	"irekon",
 	"net",
 	"pipeline",
+	"rekon",
 	"routing",
+	"vpc-manager",
```

Comment:
We'll have `mgmt` from Fredi's PR soon. This PR brings three more new crates - are we sure we need all of them to be independent crates?

Reply:
I like the idea of merging vpc-manager with mgmt